drivers/sensors/mpu60x0.c

```diff
@@ -620,6 +639,12 @@ static inline int __mpu_write_gyro_config(FAR struct mpu_dev_s *dev,
 static inline int __mpu_write_accel_config(FAR struct mpu_dev_s *dev,
                                            uint8_t afs_sel)
 {
+  if (afs_sel > 3)
+    {
+      snerr("Invalid AFS_SEL value\n");
+      return -EINVAL;
+    }
+
   uint8_t val = TO_BITFIELD(ACCEL_CONFIG__AFS_SEL, afs_sel);
```

Comment:
Please define the variable "val" at the beginning of the function, otherwise it will fail when used with older microcontrollers that only have C89 compiler.

Reply:
Thanks! Well observed